packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -193,6 +208,8 @@ class _ScrollbarState extends RawScrollbarState<Scrollbar> {
   @override
   void didChangeDependencies() {
```

Comment:
![image](https://user-images.githubusercontent.com/61075224/102600917-66a6c680-415a-11eb-8803-4bb98b0bc05d.png)

This function is all-new, but only part of the different file can be seen, which is very strange！

Reply:
Sorry I made a mistake, I shouldn't update the code, here I see the corresponding changes #72531